CONTRIBUTING.md

```diff
@@ -110,21 +116,22 @@ Go here to find documentation about [FlEXBOX component](src/components/flex/READ
 
 ## Technical Discipline
 
-#### Bumping Release Version
+#### Bumping Release Version - this part is for [maintainers](https://github.com/brainly/style-guide/blob/master/MAINTAINERS)
 
-This project uses semver versioning.
+This project uses semver versioning. We follow [SemVer](https://semver.org/):
+
+1. Bump first number - If your release breaks public API in any way - e.g. removed method, renamed class, different default parameters for method, new required parameters, removing components.
+2. Bump second number - If your release only extends public API and improves support, add new components, improves CSS. 
+3. Bump the last number - Finally, if your release only fixes a bug w/o any API changes or introducing new features bump the last number.
 
 To simplify version bumping you can use [`yarn version`](https://yarnpkg.com/lang/en/docs/cli/version/). It will patch `package.json`, create a corresponding commit, create a tag and push those to git changes.
 
 To bump the version correctly you should follow these steps:
 
-1. Wait for your PR to get accepted
-
-1. Rebase your branch
-
-1. bump version in branch 
-
-1. Merge your PR to master
+1. Merge approved, pending PR's into master.
+2. Create a branch with a bumped version according to the rules above, create tag and merge it (get approvals of course).
+3. Create proper, specific and detailed release note, with all the changes included in the release
+4. Remember to let know contributors and Brainly engineers, that new version was released and it's ready to use
```

Comment:
How would you like to achieve that? Calling in PR? wdyt?

Reply:
I guess so, that this is the best way to let know external contributors - let them know in their PR with mentioning that we are merging they changes and newest version will be available soon with theirs changes. For internal we do have slack channel

Reply:
that's okay, but maybe add that to this point? otherwise someone might look for additional contact ✌️

Reply:
@vergilius sure, on my way. thx for feedback 👌